deploy/helm/templates/statefulset.yaml

```
@@ -21,6 +21,9 @@ spec:
       {{- end }}
       labels:
         {{- include "appsmith.selectorLabels" . | nindent 8 }}
+{{- if .Values.podsLabels }}
+{{ toYaml .Values.podsLabels | indent 8 }}
+{{- end }}
```

I think you meant `podLabels` instead of `podsLabels`, right? Also, can we indent this to line up under `labels:` please? Makes it easier to read:

```suggestion
        {{- if .Values.podLabels }}
        {{- toYaml .Values.podLabels | nindent 8 }}
        {{- end }}
```